### PR TITLE
Compass scripts relocation

### PR DESCRIPTION
**Pull request by ognyvrac**
**Description**
Currently, the Compass scripts used in the ProwJobs are located in the **test-infra** repository; thus, whenever CMP needs to make changes to them, they need to create a PR here.

The scripts have been migrated to the Compass repository with this [PR](https://github.com/kyma-incubator/compass/pull/3151).

Changes proposed in this pull request:

- move scripts to the **kyma-incubator/compass** repo
- changes templates to reflect changed scripts directory

### CODE REVIEW

File: prow/jobs/incubator/compass/components/schema-migrator/compass-director-gqlgen-validate.yaml [hunk 1]
```diff
@@ -42,12 +42,15 @@ presubmits: # runs on PRs
                 type: Unconfined
               allowPrivilegeEscalation: true
             command:
-              - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/validate-compass-generic.sh"
+              - "prow/validate-compass-generic.sh"
```
Comment (nyordanoff) [inline prow/jobs/incubator/compass/components/schema-migrator/compass-director-gqlgen-validate.yaml:45]: Adapt the path here to be the new one.

File: templates/data/compass-validation-data.yaml [hunk 2]
```diff
@@ -41,14 +41,15 @@ templates:
...
+                    - "kyma_project_dir"
                   local:
                     - "jobConfig_default"
 
       - to: ../../prow/jobs/incubator/compass/components/schema-migrator/compass-director-gqlgen-validate.yaml
         localSets:
           jobConfig_default:
             path_alias: github.com/kyma-incubator/compass
-            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/validate-compass-generic.sh"
+            command: "prow/validate-compass-generic.sh"
```
Comment (nyordanoff) [inline templates/data/compass-validation-data.yaml:52]: Adapt the path here to be the new one.

**Review by nyordanoff — APPROVED** [review]

**Review by Ressetkk — APPROVED** [review]

**Pull request merged**